src/test/EndToEndRequestTest.postman_collection.json

```
@@ -1219,6 +1219,146 @@
 						"description": "Request a subset of the EOB data to ensure range requests work correctly."
 					},
 					"response": []
+				},
+				{
+					"name": "Try export with _since",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "5497fa12-aefd-44c2-9254-2beb51c52e31",
+								"exec": [
+									"// Status should be 204",
```

```suggestion
									"// Status should be 202",
```
Or maybe just remove the comment; it's clear what the test is.

---

Good catch. There is a lot of 202 vs 204 confusion in the file. Pushed a fix.